pollution_prediction/views/pollutant_views.py

```
@@ -0,0 +1,176 @@
+import os
+from configure import get_trained_model_from_gcs, Config
```

Ruff (0.8.2)

`configure.get_trained_model_from_gcs` imported but unused

Remove unused import: `configure.get_trained_model_from_gcs`

(F401)